### PR TITLE
BIP68/112/113 mining note: expand tldr and add links

### PR DESCRIPTION
**Pull request by harding**
This minor tweak adds two extra points to the TL;DR describing the nSequence and nLockTime sections below and also linking to them.  A couple other links to BIPs mentioned in the doc are also added.

CC: @btcdrak 

screenshot-127 0 0 1 4000 2016-06-21 11-16-42

### CODE REVIEW

File: _posts/en/posts/2016-06-21-csv-softfork-instructions.md [hunk 1]
```diff
@@ -15,13 +15,19 @@ excerpt: There is an ongoing soft fork of the Bitcoin consensus rules. While eve
...
+Miners must make the following changes by block #419328.  Any miners who fail to make these changes may generate invalid blocks, causing them lost revenue and possibly disrupting other users and miners on the network.
 
-2. If you manually hardcode the block version field, or use the nSequence or nLockTime fields in the coinbase transaction, you must follow the instructions carefully.
+1. Check all your nodes have been upgraded to Bitcoin Core 0.12.1 or compatible software.
+
+1. If possible, automatically set the block version from Bitcoin Core.  If not possible, set the block version to 0x20000000.  You may do this any time before block 419328.  ([Details](#hardcoded-version))
+
+1. If you generate coinbase transactions whose nSequence is not 0, ensure that the nVersion of the coinbase transaction is 1.  ([Details](#nsequence))
+
+1. If you generate coinbase transactions where the nLockTime is greater than 500000000, ensure that whatever value you set it to passes the [BIP113][] rules.  (The commonly-seen value of 0xffffffff always passes the BIP113 rules.) ([Details](#nlocktime))
```
Comment (jl2012) [inline _posts/en/posts/2016-06-21-csv-softfork-instructions.md]: "The commonly-seen value of 0xffffffff always passes the BIP113 rules." is wrong.
Should be "The commonly-seen nSequence value of 0xffffffff always passes the BIP113 rules."

Does this sound too complicated?

Comment (jl2012) [inline _posts/en/posts/2016-06-21-csv-softfork-instructions.md]: I'd prefer to simply say

If you generate coinbase transactions where the nLockTime is greater than 500000000, ensure that whatever value you set it to passes the [BIP113][] rules.  ([Details](#nlocktime))

Comment (harding) [inline _posts/en/posts/2016-06-21-csv-softfork-instructions.md]: Thanks for catching that! I was thinking about the wrong thing.  I'll change it to what you suggest.

Comment (btcdrak) [inline _posts/en/posts/2016-06-21-csv-softfork-instructions.md]: The stuff about version is not correct. The instruction is to unset the bit. The TLDR is now really too long...

Comment (harding) [inline _posts/en/posts/2016-06-21-csv-softfork-instructions.md]: Ok.  I'll close the PR then.

Comment (btcdrak) [inline _posts/en/posts/2016-06-21-csv-softfork-instructions.md]: I think adding the last two bullets is fine as 

1. If you generate coinbase transactions whose nSequence is not 0, ensure that the nVersion of the coinbase transaction is 1.  ([Details](#nsequence))

1. If you generate coinbase transactions where the nLockTime is greater than 500000000, ensure that whatever value you set it to passes the [BIP113][] rules. (Details)

**Pull request closed without merging**